synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -478,4 +487,48 @@ export class SynopsysToolsParameter {
     }
     return {};
   }
+  private setSarifReportsInputs(): Reports {
```

Comment:
Since we discussed previously why we need to use two separate methods for Blackduck and Polaris, can you update this method name to `setSarifReportsInputsForBlackduck()`  as I have implemented similar one for Polaris which is named as `setSarifReportsInputsForPolaris()`.